servicetalk-examples/http/retry/src/main/java/io/servicetalk/examples/http/retry/RetryClient.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.examples.http.retry;
+
+import io.servicetalk.http.api.HttpClient;
+import io.servicetalk.http.api.HttpResponseStatus;
+import io.servicetalk.http.netty.HttpClients;
+import io.servicetalk.http.netty.RetryingHttpRequesterFilter;
+
+import static io.servicetalk.http.api.HttpSerializers.textSerializerUtf8;
+
+/**
+ * Extends the Async "Hello World" client to immediately retry requests that get a 5XX response. Two attempts will be
```

Comment:
"Two attempts" --> "up-to two attempts"

Reply:
I was definitely unsure whether there would be two attempts or two retries, additional attempts following an initial failure. I will correct this usage but I did find it ambiguous in the API docs as to the total number of attempts that would be made, often due to whether the first attempt counted as a retry or not.

Reply:
good point, its up-to two retries, excluding the initial request.